frontend/components/AddHostsModal/PlatformWrapper/PlatformWrapper.tsx

```diff
@@ -248,7 +248,7 @@ const PlatformWrapper = ({
       // Clear message after 1 second
```

Comment:
can we update the comment too

Reply:
oofph, I accidentally left the 6 seconds I used to make it easier to screenshot the 1 second message! Fixed!